fuse/ipns/ipns_test.go

```diff
@@ -57,6 +67,38 @@ func writeFileData(t *testing.T, data []byte, path string) []byte {
 	return data
 }
 
+func verifyFile(t *testing.T, path string, data []byte) {
+	fi, err := os.Open(path)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer fi.Close()
+
+	out, err := ioutil.ReadAll(fi)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
depending on the size of the files, this may be painful for memory. could be good to use `io.RWs` instead.